src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -344,7 +344,7 @@ void Terminal::UpdateSettings(winrt::Microsoft::Terminal::Settings::ICoreSetting
 
     // If the old scrolloffset was 0, then we weren't scrolled back at all
     // before, and shouldn't be now either.
-    _scrollOffset = originalOffsetWasZero ? 0 : ::base::ClampSub(_mutableViewport.Top(), newVisibleTop);
+    _scrollOffset = originalOffsetWasZero ? 0 : static_cast<int>(::base::ClampSub(_mutableViewport.Top(), newVisibleTop));

```

Comment:
this one seems weird. should we just be usinjg `ClampSub<int>` or something?

Reply:
For whatever reason, this came up:
```
1>D:\Terminal\src\cascadia\TerminalCore\Terminal.cpp(347,104): error C2445: result type of conditional expression is ambiguous: types 'int' and 'base::internal::ClampedNumeric<short>' can be converted to multiple common types
```
It's also having trouble converting from `ClampedNumeric` to the non-clamped numeric type :/

Reply:
This is bizarre that this is needed now